### PR TITLE
Three patches for time series generation

### PR DESCRIPTION
**Pull request by mnlevy1981**
1. Added `case` argument to `env_workflow.get_job_specs()`
2. explicitly ignored last two arguments returned from
   `env_mach_specific.get_mpirun()` -- shouldn't be necessary, but I was seeing
   `ValueError: too many values to unpack` and the error went away with the
   additional `_`s
3. Remove indexing from `options.debug` in `tseries_generator`, since
   `parser.add_argument()` explicitly casts it as int; avoids
   `TypeError: 'int' object has no attribute '__getitem__'`

### CODE REVIEW

File: timeseries/template.timeseries [hunk 1]
```diff
@@ -37,7 +37,7 @@ def _main_func(description):
             "queue" : case.get_value("JOB_QUEUE", subgroup=job),
             "unit_testing" : False
             }
-        executable, mpi_arg_list = env_mach_specific.get_mpirun(case, mpi_attribs, job, overrides=overrides)
+        executable, mpi_arg_list, _, _ = env_mach_specific.get_mpirun(case, mpi_attribs, job, overrides=overrides)
```
Comment (mnlevy1981) [inline timeseries/template.timeseries:40]: I don't really understand why this change was necessary. Maybe something weird from python 2.7?

**Comment by mnlevy1981** [comment]
I needed to make these changes to run a case from `cesm2_2_beta05`, but I suspect it will break compatibility with `cesm2.1`... which probably isn't ideal if CMIP runs are making use of `--workflow timeseries`. I've got a sandbox that works for me, so I'm in no rush to see these changes back on `master`

**Comment by mnlevy1981** [comment]
Fixes #217 (I don't have permission to link the issue to this PR)

**Comment by bertinia** [comment]
@mnlevy1981 - I'm going to wait on merging this PR as it's not clear what direction the
postprocessing is going post CMIP6 and I don't want to break the current working master.

**Comment by lvankampenhout** [comment]
Regarding the indexing of the debug parameter, I agree that this should be removed. However, the proposed solution will lead to DEBUG statements when '0' is specified, since the result is stored in a list `[0]` and this list renders to `True` in a statement like: `if (debug): `. 

I propose this alternative:
```
-    parser.add_argument('--debug', nargs=1, required=False, type=int, default=0,
+    parser.add_argument('--debug', required=False, type=int, nargs='?', default=0, const=2,
```

this will work as expected in all cases, and even works when `--debug` is specified but no value is given (it will take the value of 2). See [nargs documentation](https://docs.python.org/3/library/argparse.html#nargs).

**Review by dabail10 — APPROVED** [review]

**Comment by dabail10** [comment]
@mnlevy1981 I know this is old, but could you resolve the conflicts here?
